apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -134,16 +138,12 @@ class PageTemplateModal extends Component {
 		trackDismiss( this.props.segment.id, this.props.vertical.id );
 	};
 
-	getBlocksByTemplateSlug( slug = this.state.slug ) {
-		if ( ! slug ) {
-			return [];
-		}
-
-		if ( ! this.state.blocks.hasOwnProperty( slug ) ) {
-			return [];
-		}
+	getBlocksBySlug( slug = this.state.previewedTemplate ) {
```

Comment:
Not 100% sure about renaming the method here, since `templateSlug` describes that it refers to the template slug. It worths in this context, imho.

Reply:
ok, it's using the longer wording from now on 👍